Corona-Warn-App/src/main/res/drawable/contact_diary_duration_background_selected.xml

```diff
@@ -4,4 +4,4 @@
     <corners android:radius="@dimen/radius_card" />
     <solid android:color="@color/colorSurface1" />
     <stroke android:width="2dip" android:color="@color/colorAccent" />
-</shape>
\ No newline at end of file
+</shape>
```

Comment:
Unrelated?

Reply:
I tried to revert several times, but git just says no changes detected. I do not know why this appears here.

Reply:
Accidentally selected "ignore whitespace changes"?